### DISCUSSION TITLE
How to Split schema into different files

### DISCUSSION THREAD

**Discussion by fmatuszewski** [heftapp/graphql_codegen#227] [Q&A]
Right now I keep all schema files in one folder.
But I would like to split them by features and by usage.
Fragment map to DTOs so I would like to keep them in seperate
query and mutation maps to client so I would also split it into different file

I tried to use scope, and assetPath build.yaml parameters for this but I get numerous errors
regarding visability of fragments.

**Comment by budde377** [comment]
The framework does not make any assumptions on how many schema files there are or where they are relative to each other, so if I understand you correctly, you should be able to just split your schema.
